README.md

```diff
@@ -110,35 +87,46 @@ The available options are:
                                 kept when compression is on, because of dead
                                 code removal or cascading statements into
                                 sequences.
+    --config-file <file>        Read `minify()` options from JSON file.
+    -d --define <expr>[=value]  Global definitions.
+    --ie8                       Support non-standard Internet Explorer 8.
+                                Equivalent to setting `screw_ie8: false` in `minify()`
```

Comment:
`screw_ie8` does not exist within the codebase anymore - this part of the documentation has been updated in a subsequent commit as well.